code/_globalvars/misc.dm

```diff
@@ -49,8 +49,8 @@ GLOBAL_PROTECT(topic_servers)
 GLOBAL_LIST_EMPTY(tooltips)
 
 //Should be in the form of "tag to be replaced" = list("replacement for beginning", "replacement for end")
-GLOBAL_LIST_INIT(markup_tags, list("/"  = list("<i>", "</i>"),
+GLOBAL_LIST_INIT(markup_tags, list("_"  = list("<i>", "</i>"),
 								   "**" = list("<b>", "</b>")))
 //Should be in the form of "((\\W|^)@)(\[^@\]*)(@(\\W|$)), "g"", where @ is the appropriate tag from markup_tags
-GLOBAL_LIST_INIT(markup_regex, list("/"  = new /regex("((\\W|^)_)(\[^_\]*)(_(\\W|$))", "g"),
+GLOBAL_LIST_INIT(markup_regex, list("_"  = new /regex("((\\W|^)_)(\[^_\]*)(_(\\W|$))", "g"),
```

Comment:
what's going on here? why is this change here?

Reply:
this was back when the italics tag was "/", when the original guy changed the tag to "_" he forgot to change the keys to reflect that

this doesn't break anything if that's the concern it's just for clarity's sake

Reply:
the comment above it also says 
> //Should be in the form of "tag to be replaced" = list("replacement for beginning", "replacement for end")

so fixing this respects how it's supposed to be set up

Reply:
How does this not fix or break anything? Is the code unused?

Reply:
i'm really bad at explaining things

the keys are just names meant to show what the markup tag is, I just changed the italics ones to what the tag is supposed to be, which is "_"